pip/commands/__init__.py

```diff
@@ -19,6 +20,7 @@
 
 commands_dict = {
```

Comment:
This can be changed into a dict constructor call with a list comprehension... DRY.